6_visualize/src/create_gif_tasks.R

```diff
@@ -32,23 +56,27 @@ create_gif_tasks <- function(timestep_ind, folders, view_ind, basemap_ind, storm
     },
     command = function(task_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
+
       psprintf(
         "create_storm_frame(",
-        "png_file=target_name",
+        "png_file=target_name,",
         "config=storm_frame_config,",
         "'%s',"=view_ind,
         "'%s',"=basemap_ind,
-        "'%s'"=storm_line_ind,
+        "'%s',"=river_lines_ind,
+        "'%s',"=storm_line_ind,
         "'6_vizprep/out/storm_point_[%s].rds.ind',"=cur_task$date_hour,
-        "'6_vizprep/out/precip_raster_[%s].rds.ind',"=cur_task$date_hour,
-        "'6_vizprep/out/streamdata_[%s].rds.ind')"=cur_task$date_hour,
+        "'%s')"=watermark_ind,
+        #"'6_vizprep/out/streamdata_[%s].rds.ind')"=cur_task$date_hour,
+        #"'6_vizprep/out/precip_raster_[%s].rds.ind',"=cur_task$date_hour, # need to add: precip_raster_[YYYmmdd-HH]_fun.rds.ind
         sep="\n      ")
     }
   )
 
   gif_task_plan <- scipiper::create_task_plan(
     task_names=tasks$task_name,
-    task_steps=list(gif_frame),
+    task_steps=list(point_frame, get_point_frame, gif_frame),
     add_complete=FALSE,
+    final_steps='gif_frame',
     ind_dir=folders$log)
 }
```

Comment:
Below - I think we should avoid git committing the frame .png files.

Reply:
will do